modules/installation-azure-service-principal.adoc

```diff
@@ -0,0 +1,162 @@
+subscription you want to use for your cluster:
++
+----
+$ az account list --refresh
+[
+  {
+    "cloudName": "AzureCloud",
+    "id": "9bab1460-96d5-40b3-a78e-17b15e978a80",
+    "isDefault": true,
+    "name": "Subscription Name",
+    "state": "Enabled",
+    "tenantId": "6057c7e9-b3ae-489d-a54e-de3f6bf6a8ee",
+    "user": {
+      "name": "you@example.com",
+      "type": "user"
+    }
+  }
+]
+----
+
+.. View your active account details and confirm that the `tenantID` matches
+the subscription you want to use:
++
+----
+$ az account show
+{
+  "environmentName": "AzureCloud",
+  "id": "9bab1460-96d5-40b3-a78e-17b15e978a80",
+  "isDefault": true,
+  "name": "Subscription Name",
+  "state": "Enabled",
+  "tenantId": "6057c7e9-b3ae-489d-a54e-de3f6bf6a8ee", <1>
+  "user": {
+    "name": "you@example.com",
+    "type": "user"
+  }
+}
+----
+<1> Ensure that the value of the `tenantId` parameter is the UUID of the
+correct subscription.
+
+.. If you are not using the right account, change the active subscription:
++
+----
+$ az account set -s <TenantId> <1>
+----
+<1> Substitute the value of the `TenantId` for the subscription that you want to
+use for `<TenantId>`.
+
+.. If you changed the active subscription, display your account information again:
++
+----
+$ az account show
+
+{
+  "environmentName": "AzureCloud",
+  "id": "33212d16-bdf6-45cb-b038-f6565b61edda",
+  "isDefault": true,
+  "name": "Subscription Name",
+  "state": "Enabled",
+  "tenantId": "8049c7e9-c3de-762d-a54e-dc3f6be6a7ee",
+  "user": {
+    "name": "you@example.com",
+    "type": "user"
+  }
+}
+----
+
+. Record the values of the `tenantId` and `id` parameters from the previous
+output. You need these values during {product-title} installation.
+
+. Create the service principal for your account:
++
+----
+$ az ad sp create-for-rbac --role "User Access Administrator" -role Contributor --name <service_principal> <1>
+Changing "<service_principal>" to a valid URI of "http://<service_principal>", which is the required format used for service principal names
+Retrying role assignment creation: 1/36
+Retrying role assignment creation: 2/36
+Retrying role assignment creation: 3/36
+Retrying role assignment creation: 4/36
+{
+  "appId": "8bd0d04d-0ac2-43a8-928d-705c598c6956",
+  "displayName": "<service_principal>",
+  "name": "http://<service_principal>",
+  "password": "ac461d78-bf4b-4387-ad16-7e32e328aec6",
+  "tenant": "6048c7e9-b2ad-488d-a54e-dc3f6be6a7ee"
+}
+
+----
+<1> Replace `<service_principal>` with the name to assign to the service principal.
+
+. Record the values of the `appId` and `password` parameters from the previous
+output. You need these values during {product-title} installation.
+
+. Grant additional permissions to the service principal. The service principal
+requires the legacy `Azure Active Directory Graph -> Application.ReadWrite.OwnedBy`
+permission and the `User Access Administrator` role for the cluster to
```

Comment:
Something (a verb?) is missing here: "...for the cluster to credentials for its components"